dev-java/gradle-bin/gradle-bin-4.9.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# based on: gradle-bin-3.4.1.ebuild
+
+EAPI="7"
+
+MY_PN=${PN%%-bin}
+MY_P="${MY_PN}-${PV/_rc/-rc-}"
+
+DESCRIPTION="A project automation and build tool with a Groovy based DSL"
+SRC_URI="http://services.gradle.org/distributions/${MY_P}-all.zip"
+HOMEPAGE="http://www.gradle.org/"
+LICENSE="Apache-2.0"
+SLOT="${PV}"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-arch/zip"
+RDEPEND=">=virtual/jdk-1.6"
+
+IUSE="source doc examples"
```

Comment:
This list is usually sorted alphabetically.

Reply:
Actually, what does the 'source' USE even do? Install source code? Does someone want/need that? :)
(legit question, I have no opinion on this one myself, just wondering)